lib/theme_picker_dialog.dart

```diff
@@ -1,20 +1,23 @@
 import 'package:flutter/material.dart';
-import 'package:theme_mode_handler/theme_mode_handler.dart';
 
-const themeModeOptions = [
+import 'theme_mode_handler.dart';
+
+const _themeModeOptions = [
```

Comment:
Maybe the use of an enum + extension be better to avoid errors when passing the key.

You'll access like a: ThemeModeOption.system
and access value like a: ThemeModeOption.system.value // _themeModeOption.value 

Example:

```
enum AllowancePages {
  loading,
  mainPage,
  requestCard,
  activateCard,
  trackCard,
  blockedCard,
  signPlan,
  successPage,
}

extension AllowancePagesExtension on AllowancePages {
  static const values = {
    AllowancePages.loading: 'loading',
    AllowancePages.mainPage: 'mainPage',
    AllowancePages.requestCard: 'requestCard',
    AllowancePages.activateCard: 'activateCard',
    AllowancePages.trackCard: 'trackCard',
    AllowancePages.blockedCard: 'blockedCard',
    AllowancePages.signPlan: 'signPlan',
    AllowancePages.successPage: 'successPage',
  };

  String get value => values[this];
}
```

Reply:
I don't think this is necessary as it's just an internal variable with 3 values. Would be more valid to apply an enum if it was exposed or had more values.

I don't even think people actually use this as it's a very restricted implementation but I'm keeping it to prevent larger breaking changes.

Reply:
Alright, at the first view I thought that it will be public, but alright, you did a great job.